packages/admin-rte/src/core/Rte.sc.ts

```diff
@@ -1,10 +1,15 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
 export const Root = styled.div`
     border: 1px solid ${({ theme }) => theme.rte.colors.border};
 `;
 
-export const EditorWrapper = styled.div`
+export const EditorWrapper = styled.div<{ disabled?: boolean }>`
+    ${({ disabled, theme }) =>
+        disabled &&
+        css`
+            color: ${theme.palette.text.disabled} !important;
```

Comment:
You could remove the `!important` Flag by increasing the specificity: https://styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity